music/src/core/data.ts

```diff
@@ -28,32 +28,12 @@ import * as tf from '@tensorflow/tfjs';
 import {INoteSequence, NoteSequence} from '../protobuf/index';
 
 import * as constants from './constants';
+import {DEFAULT_DRUM_PITCH_CLASSES} from './constants';
 import * as logging from './logging';
 import {Melody, MelodyControl, MelodyRhythm, MelodyShape} from './melodies';
 import * as performance from './performance';
 import * as sequences from './sequences';
 
-export const DEFAULT_DRUM_PITCH_CLASSES: number[][] = [
```

Comment:
ughhhhhhhh I think this is technically a breaking change. Can we still re-export them from here too, so that all the things that import the constants from here don't break? Sorry, this is the curse of maintaining open source 😭